web/src/components/layout/demo_layout.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { Switch, Route, withRouter } from 'react-router';
+import { useLocale } from '../locale-helpers';
+import { Spinner } from '../ui/ui';
+
+function DemoLayout() {
+  const [_, toLocaleRoute] = useLocale();
+
+  return (
+    <div>
+      <React.Suspense fallback={<Spinner />}>
```

Comment:
Nitpick - I know it's used this way elsewhere in the codebase, but I tend to think it's cleaner to separately import the Suspense Component and then just use it as `<Suspense>`